### PR TITLE
Improve the NDEF parsing algorithms

### PR DESCRIPTION
**Pull request by zolkis**
Signed-off-by: Zoltan Kis <zoltan.kis@intel.com>

***
<a href="https://pr-preview.s3.amazonaws.com/zolkis/web-nfc/pull/323.html" title="Last updated on Aug 30, 2019, 4:58 PM UTC (3f59026)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/323/6ab55b8...zolkis:3f59026.html" title="Last updated on Aug 30, 2019, 4:58 PM UTC (3f59026)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3336,7 +3336,7 @@ <h3>Receiving and parsing content</h3>
         equal to an empty ordered map.
       </li>
       <li>
-        If |ndefRecord|'s <a>PAYLOAD field</a> is not present, set |record|s mediaType to |mimeType|, set |record|.[[\PayloadData]] to <code>null</code> and return |record|.
+        If |ndefRecord|'s <a>PAYLOAD field</a> is not present, set |record|s mediaType to |mimeType|, set |record|.[[\PayloadData]] to `undefined` and return |record|.
```
Comment (kenchris) [inline index.html:3339]: Can we wrap these lines a bit?

**Review by kenchris — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (zolkis) [reply index.html:3339]: Ah, I tend to forget we do manual breaks in Web NFC, for PR-review friendly diffs. 
Let's make that over all the spec in a future editorial PR.

**Comment by kenchris** [comment]
These is this tidy comment that can fix it, but I always have problem getting it installed correctly... maybe check with @anssiko

**Comment by anssiko** [comment]
Grab `tidy` from http://binaries.html-tidy.org/ or https://github.com/htacg/tidy-html5

`tidy-config.txt` (for example, adapt to your needs):
```
char-encoding: utf8
indent: yes
wrap: 80
tidy-mark: no
```
`$ tidy -config tidy-config.txt -output output input`

**Comment by zolkis** [comment]
Thanks. This also did it for me:
```
$ sudo dnf install tidy
```

We'll have changes done by tidy all the time, but still better than manually doing the same.

**Pull request merged**